Annotato/Annotato/ViewModels/Document/DocumentViewModel.swift

```diff
@@ -163,6 +165,16 @@ extension DocumentViewModel {
         annotationViewModel?.receiveDelete()
         annotations.removeAll(where: { $0.model.id == deletedAnnotation.id })
     }
+
+    func receiveCreatedOrUpdatedAnnotation(createdOrUpdatedAnnotation: Annotation) {
+        if annotations.contains(where: { $0.id == createdOrUpdatedAnnotation.id }) {
```

Comment:
Not sure if this works, because the annotationViewModel array only contains non deleted stuff? So imagine it doesn't contain an entity, but it's actually in the model's array. Then it's an incorrect call to receiveNewAnnotation?

Reply:
Yes you're right, I changed
Will only be able to test during integration though